app/js/lib/foxbox/event-dispatcher.js

```diff
@@ -169,7 +157,8 @@ export default class EventDispatcher {
 
   /**
    * Removes all registered listeners for the specified event.
-   * @param {string} eventName Name of the event to remove all listeners for.
+   *
+   * @param {string} [eventName] Name of the event to remove all listeners for.
```

Comment:
It works for me on WebStorm but I don't think I set up ESLint properly. If it's a bug maybe we should fill a ticket in WebStorm or ESLint.